Dockerfile.image

```diff
@@ -0,0 +1,68 @@
+# ARGS go first if used on FROM
+ARG OPERATOR_IMAGE=quay.io/costoolkit/elemental-operator:v0.3.0
+ARG SYSTEM_AGENT_IMAGE=rancher/system-agent:v0.2.9
```

Comment:
Isn't better to use the `latest` released version instead of a fixed one?

Reply:
Not really, I tried to use the versions that are being used on OBS currently. Maybe once we got tests we can move this into a latest tag or maybe have an extra job that uses the latest tags for everything so we know if something breaks before it's changed in OBS, kind of a staging job